D1673/P1673.md

```diff
@@ -9158,6 +9153,8 @@ not the output matrix -- is Hermitian. <i>-- end note]</i>
 
   * [2.3]{.pnum} `C.extent(1)` equals `E.extent(1)` (if applicable).
 
+  * [2.4]{.pnum} The input matrix is Hermitian.
```

Comment:
Please see above note (the input matrix isn't Hermitian, it's interpreted as such; this should be covered by existing wording and changes in other PRs).